KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -59,9 +60,18 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                // Dotnet("test" + testArgs, projectFolder);
+                if (!IsLinux ||
+                    string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO) ||
+                    string.Equals(KOREBUILD_TEST_SKIPMONO, "0") ||
+                    string.Equals(KOREBUILD_TEST_SKIPMONO, "false"))
+                {
+                    // Dotnet("test" + testArgs, projectFolder);
+                }
             }
-            else if (!IsLinux || !string.IsNullOrEmpty(KOREBUILD_TEST_DNXCORE))
+            else if (!IsLinux ||
+                     (!string.IsNullOrEmpty(KOREBUILD_TEST_DNXCORE) &&
+                      !string.Equals(KOREBUILD_TEST_DNXCORE, "0") &&
+                      !string.Equals(KOREBUILD_TEST_DNXCORE, "false")))
```

Comment:
Conditions are rather odd. Where else do we consider "anything but `0` and `false`" as true? Use `string.IsNullOrEmpty()` to be consistent with existing `KOREBUILD_TEST_DNXCORE` checks. Or switch both checks to treat only `1` and `true` as true, doing part of #326.